ecs-deploy/locals.tf

```diff
@@ -6,76 +6,66 @@ data "aws_caller_identity" "current" {}
 data "aws_region" "current" {}
 
 locals {
+  name       = var.cluster_name != null ? var.cluster_name : "${var.project}-${var.environment}${var.regional ? "-${var.region}" : ""}"
+  account_id = data.aws_caller_identity.current.account_id
+  region     = data.aws_region.current.name
```

Comment:
where is the region read from? Assume I have set a specific region the terraform/aws provider, but my local shell has a different `AWS_REGION` env var set would that result in a conflict?

Reply:
it is read from the aws profile which run the terraform script
1. If you have aws provider and specify the region -> this region would be used
2. If you don't have aws provider but you have env var -> it will use the default profile